src/batcher/solve_pair.rs

```diff
@@ -1,16 +1,435 @@
 use crate::models::order::Order;
 use crate::models::solution::Solution;
 use anyhow::Result;
-use web3::types::U256;
+use web3::types::{U256, H256, Address};
+
+
+pub enum OrderPairType {
```

Comment:
I am wondering whether we need this particular enum: 
Since we will fill both orders fully, it is probably not really needed. But on the other hand, it would be good if the: sell_orders_token0 would always have the orders in the batch, where the sellvolume exceeds the other sellvolume, in our formula 'A(T0)>pA(T1)`, so that the smart contract does not need too invert the orders beforehand.

I am not sure, whether we want to considered this already in this PR. As you think, @twalth3r